openpower/package/openpower-pnor-p10/openpower-pnor-p10.mk

```diff
@@ -146,6 +146,8 @@ define OPENPOWER_PNOR_P10_UPDATE_IMAGE
             PATH=$(HOST_DIR)/usr/sbin:$(PATH) $(HOST_DIR)/usr/bin/generate-ubi \
                 $(XML_VAR).pnor.squashfs.tar ;\
             $(INSTALL) $(STAGING_DIR)/pnor.$(XML_VAR)/$(XML_VAR).pnor.ubi.mtd $(BINARIES_DIR) ;\
+            cp $(STAGING_DIR)/hostboot_build_images/hbicore.syms $(PNOR_SCRATCH_DIR)/HBICORE_SYMS.ipllid ;\
```

Comment:
Wondering if we should have done something like :

    $(INSTALL) -m 0644 -D $(PNOR_SCRATCH_DIR)/HBICORE_SYMS.ipllid $(STAGING_DIR)/hostboot_build_images/hbicore.syms

for these two, to control the end permissions and such